stories/core/Table.stories.tsx

```diff
@@ -210,11 +207,23 @@ export const Selectable: Story<TableProps> = (args) => {
             Header: 'Click',
             width: 100,
             Cell: (props: CellProps<{ name: string; description: string }>) => {
-              const onClick = () => onClickHandler(props);
               return (
-                <a className='iui-anchor' onClick={onClick}>
-                  Click me!
-                </a>
+                <span
+                  style={{
+                    display: 'flex',
+                    width: '100%',
+                    height: '100%',
+                    alignItems: 'center',
+                  }}
+                  onClick={(e) => e.stopPropagation()} // prevent row selection when this cell is clicked
+                >
+                  <a
+                    className='iui-anchor'
+                    onClick={() => action(props.row.original.name)()}
```

Comment:
You can write like this:
```suggestion
                    onClick={action(props.row.original.name)}
```

Reply:
Is it meant to be written like that? 

https://jakearchibald.com/2021/function-callback-risks/

Reply:
`action(...)` returns a function that accepts any args so I guess it is not bad doing that 🤷‍♂️